### PR TITLE
ip tagging: coverage

### PR DESCRIPTION
**Pull request by junr03**
@lyft/network-team

### CODE REVIEW

File: test/common/http/filter/ip_tagging_filter_test.cc [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+#include "common/http/headers.h"
+
+#include "test/mocks/http/mocks.h"
+#include "test/test_common/utility.h"
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+namespace Envoy {
+using testing::_;
```
Comment (mattklein123) [inline test/common/http/filter/ip_tagging_filter_test.cc]: nit: move usings outside of Envoy namespace.

Comment (junr03) [inline test/common/http/filter/ip_tagging_filter_test.cc]: I was following from the other test files. Seems that when we moved to the enclosing Envoy namespace all the usings got trapped inside.

Comment (mattklein123) [inline test/common/http/filter/ip_tagging_filter_test.cc]: I have been fixing them as I find them.

Comment (junr03) [inline test/common/http/filter/ip_tagging_filter_test.cc]: good to know. I will fix this one.

**Pull request merged**